code/game/gamemodes/gameticker.dm

```diff
@@ -422,6 +422,7 @@ var/datum/controller/gameticker/ticker
 			if(player.mind.assigned_role != "MODE")
 				job_master.EquipRank(player, player.mind.assigned_role, 0)
 				EquipCustomItems(player)
+			player.apeify()
```

Comment:
Wouldn't it be better to just move the contents of the proc here? It's just a monkeyize with extra checks

Reply:
nope